ketcher-autotests/tests/File-Management/Open-And-Save-Files/MOL-Files/mol-files-export-import-query-features.spec.ts

```diff
@@ -0,0 +1,1592 @@
+/* eslint-disable prettier/prettier */
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  waitForPageInit,
+  openFileAndAddToCanvasAsNewProject,
+  saveToFile,
+  selectTopPanelButton,
+  TopPanelButton,
+  clickOnFileFormatDropdown,
+  pressButton,
+  moveMouseAway,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test('1. Aromaticity: User can export and load back molecules with Aromaticity query feature to Mol file. Feature values remain in place.', async ({
+    page,
+  }) => {
+    /*
+        Test case: https://github.com/epam/ketcher/issues/3860 - Case 1
+        Description: User can export and load back molecules with "Aromaticity" query feature to Mol file. Feature values remain in place.
+        1. Clear canvas
+        2. Load Aromaticity.ket (could be obtained from here: Aromaticity.zip (unzip first))
+        3. Validate canvas
+        4. Export canvas to Mol file
+        5. Clear canvas
+        6. Load Mol file
+        7. Validate canvas
+        */
+    await openFileAndAddToCanvasAsNewProject(
+      'KET/Query-Features/Aromaticity.ket',
```

Comment:
In general, I have no comments; the only thing in future tests is that the names of test files should be given with a small letter) (for example aromaticity.ket)

Reply:
Ok, no problem next time